src/pyscipopt/scip.pxd

```diff
@@ -1671,6 +1712,12 @@ cdef extern from "scip/scip_tree.h":
     SCIP_NODE* SCIPgetBestNode(SCIP* scip)
     SCIP_NODE* SCIPgetBestboundNode(SCIP* scip)
     SCIP_RETCODE SCIPrepropagateNode(SCIP* scip, SCIP_NODE* node)
+    int SCIPgetFocusDepth(SCIP *scip)
+    int SCIPgetPlungeDepth(SCIP *scip)
+    int SCIPgetEffectiveRootDepth(SCIP* scip)
+    int SCIPgetNNodesLeft(SCIP* scip)
+    int SCIPgetCutoffdepth(SCIP* scip)
+    int SCIPgetRepropdepth(SCIP* scip)
```

Comment:
Could you explain where this is used for your intended functionality?

Reply:
It has none. Sorry. I cleaned it up more. It should be good now :)